default.nix

```diff
@@ -22,6 +22,7 @@ in
 , withNuma   ? nixpkgs.stdenv.isLinux
 , withDtrace ? nixpkgs.stdenv.isLinux
 , withGrind ? true
+, fixWSL2 ? false # fix time-compat build issues for WSL2
```

Comment:
Wouldn't it make sense to set the default according to the environment?

Like `withDtrace ? nixpkgs.stdenv.isLinux`, but for WSL2 if that is possible? :thinking: